File: online_bookshop_API/online_bookshop_API/Model/Book.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace online_bookshop_API
{
    public partial class Book
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Summary { get; set; }
        public string Isbn { get; set; }
        public byte[] Image { get; set; }
        public string Author { get; set; }
        public DateTime Year { get; set; }
        public decimal Price { get; set; }
        public int NumPages { get; set; }
        public int Quantity { get; set; }
        public int NumPurchases { get; set; }
    }
}

File: BookshopWebApp/src/app/Model/Book.ts
export class Book {
    id?: number = 0;
    title: string = "";
    description: string = "";
    summary: string = "";
    iSBN: string = "";
    image?: Blob = new Blob() ;
    author: string = "";
    year: Date = new Date() ;
    price: number = 0;
    numPages: number = 0;
    numPurchases: number = 0;
    quantity: number = 0;
  }